### DISCUSSION TITLE
Aceforce2 key mapping (valorant mobile clone)

### DISCUSSION THREAD

**Discussion by noobmaster60nine** [barry-ran/QtScrcpy#1003] [Q&A]
Please share key mapping for aceforce2 default control layout thanks

I have screenshot of my controls
![Screenshot_2024-07-11-06-57-38-514_com proximabeta mf aceforce2](https://github.com/barry-ran/QtScrcpy/assets/175264609/48bb6cb5-7056-4f5b-9f36-3dbe6b56df77)

**Comment by w4po** [comment]
Hi, Sorry for the late response but I've just made a visual key mapper for QtScrcpy: https://github.com/w4po/ScrcpyKeyMapper
You can now create your own <3
